hack/machineset.sh

```diff
@@ -185,7 +185,7 @@ get_vsphere_ms() {
 
```

Comment:
You may need to re-open https://github.com/openshift/release/pull/26435

Reply:
For sure